jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-1-week/Jenkinsfile

```diff
@@ -5,7 +5,7 @@ node {
 
 pipeline.single_pipeline([
         job_name: 'oss-latest-1-week',
-        gatling_simulation_config: '../simulation-runner/config/scenarios/foss25x-medium-1250-1-week.json',
+        gatling_simulation_config: '../simulation-runner/config/scenarios/foss5x-medium-1250-1-week.json',
```

Comment:
I think you need to add `jruby_jar: "/opt/puppetlabs/server/apps/puppetserver/jruby-9k.jar",` to get jruby9k in here.  not sure if we want a 1.7 and 9k version for comparison?

Reply:
I looked at the 9k configuration for comparison, saw that file also specifies agent_version: https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby9k/Jenkinsfile#L13. I assume we want that? No idea what the behavior is if it's not specified.

Would 1.7 data be useful?

Reply:
agent_version sounds good to me. i think having 1.7 data to compare against will be useful yes.

Reply:
i guess we probably want jit on also then for the 9k work?

Reply:
we probably eventually want a job with max-requests-per-instance set to 100k like in PE, but until we've picked up the jruby-utils work to offset jruby expiration we'll get a lot of KOs in those runs which would make the data less useful.

Reply:
Cool cool, so i'll update this to have separate jobs for each version of jruby, and we'll see how it does without max-requests-per-instance.

Reply:
https://www.youtube.com/watch?v=fC_q9KPczAg